### PR TITLE
Add a footer for the template.

### PR DESCRIPTION
**Pull request by FledgeXu**

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -388,3 +388,9 @@ blockquote {
     border-color: var(--border-color);
     color: var(--foreground-secondary-color);
 }
+
+footer {
+    font-size: 0.9em;
+    color: #999;
```
Comment (livid) [inline assets/style.css]: Each new change to the template should support both light and dark modes. That is why all of our color values are CSS variables. After I merged your last PR for blockquote, I changed all those CSS values to CSS variables and made sure they looked good for dark mode too.

**Review by livid — CHANGES_REQUESTED** [review]

File: assets/style.css (hunk 1 above)
Comment (FledgeXu) [reply assets/style.css]: Ok, I will fix it later.

Comment (FledgeXu) [reply assets/style.css]: I tested this color, it looks fine both in light and dark mode.
image
image

**Comment by livid** [comment]
I am working on this aligning issue.

Screenshot 2022-12-09 at 4 46 21 PM

**Comment by FledgeXu** [comment]
I noticed this aligning issue. Because I found I need to change the `blog.html`, so I didn't fixed it.

**Pull request merged**